packages/calcite-components/src/components/action/action.scss

```diff
@@ -7,6 +7,8 @@
  * @prop --calcite-action-background-color: Specifies the component's  background color.
  * @prop --calcite-action-background-color-hover: Specifies the component's background color when hovered or focused.
  * @prop --calcite-action-background-color-pressed: Specifies the component's background color when active.
+ * @prop --calcite-action-corner-radius-start-end: Specifies the component's corner radius start end.
+ * @prop --calcite-action-corner-radius-end-end: Specifies the component's corner radius end end.
```

Comment:
@alisonailea could you chime in on this bit suggested by @jcfranco:

I think we'd want the following since Action is used across different components and in different contexts:
 
`--calcite-action-corner-radius` – for most, simple cases
`--calcite-action-corner-radius-<start|end>-<start|end>` – for finer-grained cases (e.g., close buttons)

For the latter, we could keep them internal in case these are only needed for close buttons and consider adding a close button component.